File: cpp/ZeroJudge/a006.cpp
#include<iostream>
#include<cmath>

using namespace std;

int main(){
	int a, b, c;
	cin >> a >> b >> c;
	int D = b * b  - 4 * a * c;
	if(D > 0){
		cout << "Two different roots x1=" << (-b+int(pow(D,0.5)))/(2*a) << " , x2=" << (-b-int(pow(D,0.5)))/(2*a) << "\n";
	}else if(D == 0){
		cout << "Two same roots x=" << -b/(2*a) << "\n";
	}else{
		cout << "No real root\n";
	}
}

File: python/ZeroJudge/a006.py
"""
https://zerojudge.tw/ShowProblem?problemid=a006
"""

a, b, c = [int(x) for x in input().split()]
d: int = b**2 - 4 * a * c
if d == 0:
    print(f'Two same roots x={int(-b / (2 * a))}')
elif d > 0:
    print(f'Two different roots x1={int((-b + d**0.5) / (2 * a))} , x2={int((-b - d**0.5) / (2 * a))}')
else:
    print('No real root')